src/commands/init/ai-rules.ts

```diff
@@ -0,0 +1,447 @@
+    throw new Error(`Failed to configure Windsurf MCP: ${error instanceof Error ? error.message : 'Unknown error'}`)
+  }
+}
+
+// Generic MCP configuration display
+const showGenericMcpConfig = (config: string, ideName: string): void => {
+  log(`\n${chalk.yellow('📋 Manual Configuration Required')}`)
+  log(`Please add the following configuration to your ${ideName} settings:`)
+  log(`\n${chalk.gray('--- Configuration ---')}`)
+  log(config)
+  log(`${chalk.gray('--- End Configuration ---')}\n`)
+}
+
+// Trigger IDE-specific MCP configuration
+const triggerMcpConfiguration = async (ide: ConsumerConfig, projectPath: string): Promise<boolean> => {
+  log(`\n${chalk.blue('🔧 MCP Configuration for')} ${chalk.cyan(ide.presentedName)}`)
+
+  const { shouldConfigure } = await inquirer.prompt<{ shouldConfigure: boolean }>([
+    {
+      type: 'confirm',
+      name: 'shouldConfigure',
+      message: `Would you like to automatically configure MCP server for ${ide.presentedName}?`,
+      default: true,
+    },
+  ])
+
+  if (!shouldConfigure) {
+    log(chalk.gray('Skipped MCP configuration. You can set it up manually later.'))
+    return false
+  }
+
+  try {
+    const config = generateMcpConfig(ide)
+
+    // IDE-specific configuration actions
+    switch (ide.key) {
+      case 'vscode':
+        await configureMcpForVSCode(config, projectPath)
+        break
+      case 'cursor':
+        await configureMcpForCursor(config, projectPath)
+        break
+      case 'windsurf':
+        await configureMcpForWindsurf(config, projectPath)
+        break
+      default:
+        showGenericMcpConfig(config, ide.presentedName)
+    }
+
+    log(`${chalk.green('✅')} MCP configuration completed for ${chalk.cyan(ide.presentedName)}`)
+    return true
+  } catch (error) {
+    log(`${chalk.red('❌')} Failed to configure MCP: ${error instanceof Error ? error.message : 'Unknown error'}`)
+    return false
+  }
+}
+
+// Helper functions reused from ai-start.ts
+const decodeHash = (hash: string): string => {
+  try {
+    return atob(hash)
+  } catch (error) {
+    throw new Error(`Failed to decode hash: ${error instanceof Error ? error.message : 'Invalid base64 or URL'}`)
+  }
+}
+
+const fetchProjectInfo = async (url: string): Promise<ProjectInfo> => {
+  try {
+    const response = await fetch(url, {
+      headers: {
+        'content-type': 'text/plain',
+      },
+    })
+
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${String(response.status)}`)
+    }
+    const data = (await response.text()) as unknown as string
+    const parsedData = JSON.parse(data) as unknown as ProjectInfo
+    return parsedData
+  } catch (error) {
+    throw new Error(`Failed to fetch project information: ${error instanceof Error ? error.message : 'Unknown error'}`)
+  }
+}
+
+const getRepoUrlFromProjectId = async (api: NetlifyAPI, projectId: string): Promise<string> => {
+  try {
+    const SiteInfo = (await api.getSite({ siteId: projectId })) as SiteInfo
```

Comment:
This is not a class?

```suggestion
    const siteInfo = (await api.getSite({ siteId: projectId })) as SiteInfo
```